kinto_http/session.py

```diff
@@ -68,6 +73,12 @@ def request(self, method, endpoint, data=None, permissions=None,
         retry = self.nb_retry
         while retry >= 0:
             resp = requests.request(method, actual_url, **kwargs)
+            retry_after = resp.headers.get("Backoff")
```

Comment:
nit: `retry_after` is confusing here. `backoff_seconds` or `backoff_duration` would be better